### ISSUE TITLE
Sympa Accessibility Issue - 23 - Back to Top button is keyboard inaccessible

### ISSUE THREAD

**Issue by rm1-uiuc** [sympa-community/sympa#1775]
The University of Illinois at Urbana-Champaign, Technology Services Accessibility Testing team conducted an Accessibility Review of Sympa on September 25, 2023 and found 26 issues that need to be fixed to make the interface accessible for visual impaired users.

Version
-------
6.2.72

Installation method
-------------------
Source Package

Expected behavior
-----------------
Back to Top button needs to be read by the screen reader and should be accessible by the keyboard.

Actual behavior
---------------
Back to Top button not keyboard accessible

Steps to reproduce
------------------
Sympa Accessibility Issue - 23 - was determined by an Automation Tool Evaluation using AInspector and Siteimprove Accessibility Checkers as automated accessibility evaluation tools.

After running these automation tools the following issue was identified:

Back to Top button is keyboard inaccessible.

[2.1.1 Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html)

![image](https://github.com/sympa-community/sympa/assets/144282683/7fae2ae2-864a-4d41-984e-60eefeb0a601)

Fig 24 Back to Top button not keyboard accessible

Additional information
----------------------
Attached is the Accessibility Review conducted on Sept 25,2023
[Tech Services Review - Accessibility Evaluation Report for Sympa.pdf](https://github.com/sympa-community/sympa/files/13562523/Tech.Services.Review.-.Accessibility.Evaluation.Report.for.Sympa.pdf)

**Comment by ikedas** [comment]
Hi @rm1-uiuc ,

What do you suppose the countermeasure for _keyboard inaccessibility_ ?
- Tabindexing seems irrelevant to this issue.
- We might define any keyboard-shortcut using Control key or such, however it seems unintuitive.

**Comment by ikedas** [comment]
If there are no good suggestion, this issue woiuld be better to be closed with [wontfix] tag for now.
